akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala

```diff
@@ -87,13 +73,20 @@ class StreamTcpDocSpec extends AkkaSpec {
         val out = UndefinedSink[ByteString]
 
         val welcomeMsg =
-          s"""|Welcome to: ${connection.localAddress}!
-              |You are: ${connection.remoteAddress}!""".stripMargin
+          s"Welcome to: ${connection.localAddress}!" +
+            s"You are: ${connection.remoteAddress}!\n"
```

Comment:
If this is intended to be one line then it would be better to write it on one as well. If the line length limit hits then `import connection._` or similar.